pkg/operator/cluster/cluster.go

```diff
@@ -110,6 +126,23 @@ func (c *Cluster) CreateInstance() error {
 	return nil
 }
 
+func (c *Cluster) Monitor(stopCh <-chan struct{}) {
+	for {
+		select {
+		case <-stopCh:
+			logger.Infof("Stopping monitoring of cluster %s", c.Spec.Namespace)
+			return
+
+		case <-time.After(healthCheckInterval):
+			logger.Debugf("checking health of mons")
+			err := c.mons.CheckHealth()
+			if err != nil {
+				logger.Infof("failed to check mon health. %+v", err)
```

Comment:
info seems right to me. the cluster can keep functioning just fine for a long time without the health checks